src/widgets/SearchBar.js

```diff
@@ -10,8 +10,9 @@ import PropTypes from 'prop-types';
 
 import { StyleSheet, TextInput, View, TouchableOpacity } from 'react-native';
 import { MagnifyingGlass, Cancel } from './icons';
-import { SUSSOL_ORANGE, APP_FONT_FAMILY } from '../globalStyles/index';
+import { APP_FONT_FAMILY } from '../globalStyles/index';
 import { debounce } from '../utilities/index';
+import { WHITE } from '../globalStyles/colors';
```

Comment:
This will change the colour of the component for the whole app, rather than just within this modal. The default being orange is correct, you need to pass through the colour white as a prop